src/main.rs

```diff
@@ -357,6 +372,8 @@ fn main() {
     // Parse both input files.
     let ast_base = parse_file::<ast::FromNodeId>(&args.arg_base_file, &lexer1, &parser1);
     let ast_diff = parse_file::<ast::ToNodeId>(&args.arg_diff_file, &lexer2, &parser2);
+    let ast_base_clone = ast_base.clone();
```

Comment:
Why do we clone the two ASTs? This will be expensive, so if this is something that we really need to do for Chawathe98, let's move it so that we only clone if we're using that algorithm.